cysync/src/pages/mainApp/sidebar/transaction/index.tsx

```diff
@@ -375,7 +375,38 @@ const Transaction = () => {
         open={showTxn !== null}
         handleClose={() => setShowTxn(null)}
         isClosePresent
-        restComponents={<TransactionDialog txn={showTxn} />}
+        restComponents={
+          <TransactionDialog
```

Comment:
Why not? 
```
<TransactionDialog 
txn={showTxn? showTxn: null }
>
```
instead of doing individual mapping

Reply:
Because it contains some default database entries such as _id, createdAt, __v etc. These extra properties results in warning on the console.

There were 2 options.
1. Acknowledge the database default values and define them in types
2. Pass only the required details.

I did not prefer the 1st one because the database specific variables might change in future.

Reply:
okay,